docs/resources/rule_uploads.md

```diff
@@ -28,52 +28,52 @@ resource "wallarm_rule_uploads" "allow_markup_in_body" {
 
 ## Argument Reference
 
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
-* file_type - (Required) File type to allow. Possible values: `docs`, `html`, `images`, `music`, `video`.
-* `point` - (Required) Request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `client_id` - (optional) ID of the client to apply the rules to. The value is required for [multi-tenant scenarios][2].
+* `action` - (optional) rule conditions. Possible attributes are described below.
+* file_type - (**required**) file type to allow. Possible values: `docs`, `html`, `images`, `music`, `video`.
```

Comment:
why not a code?

Reply:
What code? These are the only values supported by the API. They may have different names in the UI.